examples/oneapi/cpp/source/svc/svm_thunder_dense_batch.cpp

```diff
@@ -0,0 +1,113 @@
+/*******************************************************************************
+* Copyright 2020 Intel Corporation
+*
+* Licensed under the Apache License, Version 2.0 (the "License");
+* you may not use this file except in compliance with the License.
+* You may obtain a copy of the License at
+*
+*     http://www.apache.org/licenses/LICENSE-2.0
+*
+* Unless required by applicable law or agreed to in writing, software
+* distributed under the License is distributed on an "AS IS" BASIS,
+* WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+* See the License for the specific language governing permissions and
+* limitations under the License.
+*******************************************************************************/
+
+#include <iomanip>
+#include <iostream>
+
+#include "oneapi/dal/data/table.hpp"
+#include "oneapi/dal/data/accessor.hpp"
+#include "oneapi/dal/algo/svm.hpp"
+
+using namespace oneapi;
+
+std::ostream &operator <<(std::ostream& stream, const dal::table& table) {
```

Comment:
Move it to `utils` folder, see how it is done in classical examples

Reply:
Sure, I'm only afraid of conflicts. For good I’ll try to do it in a separate PR.